packages/stargate/src/stargateclient.ts

```diff
@@ -80,6 +78,29 @@ export class StargateClient {
     }
   }
 
+  /**
+   * Queries all balances for all denoms that belong to this address.
+   *
+   * Uses the grpc queries (which iterates over the store internally), and we cannot get
+   * proofs from such a method.
+   */
+  public async getUnverifiedAllBalances(address: string): Promise<readonly Coin[]> {
```

Comment:
Suggest `getAllBalancesUnverified`, otherwise it sounds really weird. (And using `Unverified` generally as a suffix.)